### DISCUSSION TITLE
Bright red in logs - [MusicPlayerBackgroundTask][Timestamp]Player error code 0: Source error

### DISCUSSION THREAD

**Discussion by jcdick1** [UnicornsOnLSD/finamp#472] [General]
My stock JF Android client plays music just fine.  Lately, Finamp though shows it is trying to play, but no advancement in the track and no sound.

Meanwhile, the above showed up in the logs the last time I tried to use it.  The stack trace was Null.

Anyone know what that may be about?

JF 10.8.10
FInamp 0.6.14

Thanks!

**Comment by UnicornsOnLSD** [comment]
What format is the music? That error means that the player failed to start playing because it couldn't recognise the file.
